docs/best_practices/tool_xml.rst

```diff
@@ -117,7 +117,7 @@ It's convenient to do something like:
 
 .. code:: console
 
-    ln -sfn $input\_fasta tmp.fa;
+    ln -sfn "${input}"\_fasta tmp.fa;
```

Comment:
Was the original parameter ``input`` or ``input_fasta``? i.e. are the curly braces in the right place?

Also isn't the closing quote in the wrong place? e.g. ``ln -sfn "$input\_fasta" tmp.fa;``